Corzbank.Api/Controllers/UsersController.cs

```diff
@@ -32,20 +32,46 @@ public async Task<IActionResult> GetUserById(Guid id)
             return Ok(result);
         }
 
-        [HttpPost]
+        [HttpPost("register")]
         public async Task<IActionResult> RegsiterUser([FromBody] UserModel user)
         {
             var result = await _userService.RegisterUser(user);
 
-            return Ok(result);
+            if (result != null)
+            {
+                foreach (var errorList in result)
+                {
+                    foreach (var error in errorList.Errors)
+                    {
+                        ModelState.AddModelError(error.Code, error.Description);
+                    }
+                }
```

Comment:
Do we really need that for UI? If so, create a separate method for that. That logic can be used in different places

Reply:
resolved